clients/drcachesim/simulator/cache.cpp

```diff
@@ -37,13 +37,15 @@
 bool
```

Comment:
It's good to have an issue for a noticeable feature like this so I filed #3037.  Please include it in the title (with an i) and commit message (Fixes or Issue: per the workflow guide) when merging.

Reply:
Done. Thanks for filing the feature request.